src/OrchardCore.Modules/OrchardCore.AdminMenu/Assets/js/AdminNodeIconPicker.js

```diff
@@ -32,12 +33,10 @@ var iconPickerVue = new Vue({
                 $('#' + self.targetIconTag).replaceWith('<i id="' + self.targetIconTag + '" class="'+ selected + '"></i>')                
             }
 
-            var iconPickerModal = new bootstrap.Modal($("#iconPickerModal"), {
-                backdrop: 'static',
-                keyboard: false
-            });
-
-            iconPickerModal.show();
```

Comment:
Why the model was shown, after the update you just hide it?

Reply:
When selecting an icon with a mouse click it selects the icon and hides the modal to show the selected icon in the dropdown button that is on the page. I don't know why it was showing the modal instead.

Reply:
I may try this, unless you on rush to accept this a quick fix ;)

Reply:
Meeting is in 2 hours. It needs to be merged for 1.4. If you think the code is bad, I won't argue, I did this really quick and I know that using the Vue data to pass the object is maybe not common but it works :smile:.

Reply:
> Meeting is in 2 hours

As expected ;) could you please check it in ore than one browse, if it works we can accepted for now, unless someone knows the reason why the modal is showing

I may have a look to this, probably after you merge it ;)

Reply:
Also, remember that now Bootstrap 5 creates a new instance of a Modal everytime we do a `new bootstrap.Modal`

Reply:
Works in Chrome, Edge (Chromium), Firefox.

Reply:
> So it can create multiple Modals on the same DOM element which is wrong in that case.

May be in this case, but I expect the modal will be hidden when you click the backdrop layer

Reply:
It doesn't because we make the backdrop "static" like a modal it expects that a value is selected.

Reply:
https://user-images.githubusercontent.com/3228637/171234709-b966dcbe-ab1c-454f-95f0-969e6a24e604.mp4

Removing the "static" setting on the modal now closes the modal when clicking the backdrop.